WordCountException: Error: 13 is an invalid word count.

``` ruby
06-18 22:21:21.729 13022 13022 E AndroidRuntime: java.lang.AssertionError: cash.z.ecc.android.bip39.Mnemonics$WordCountException: Error: 13 is an invalid word count.
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at com.stevesoltys.seedvault.ui.recoverycode.RecoveryCodeViewModel.validateCode(RecoveryCodeViewModel.kt:65)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at com.stevesoltys.seedvault.ui.recoverycode.RecoveryCodeInputFragment.done(RecoveryCodeInputFragment.kt:141)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at com.stevesoltys.seedvault.ui.recoverycode.RecoveryCodeInputFragment.access$done(RecoveryCodeInputFragment.kt:41)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at com.stevesoltys.seedvault.ui.recoverycode.RecoveryCodeInputFragment$onViewCreated$2.onClick(RecoveryCodeInputFragment.kt:122)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.view.View.performClick(View.java:7729)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.view.View.performClickInternal(View.java:7706)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.view.View.-$$Nest$mperformClickInternal(Unknown Source:0)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.view.View$PerformClick.run(View.java:30484)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:959)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:100)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:232)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:317)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8501)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: Caused by: cash.z.ecc.android.bip39.Mnemonics$WordCountException: Error: 13 is an invalid word count.
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at cash.z.ecc.android.bip39.Mnemonics$MnemonicCode.validate(Mnemonics.kt:108)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	at com.stevesoltys.seedvault.ui.recoverycode.RecoveryCodeViewModel.validateCode(RecoveryCodeViewModel.kt:63)
06-18 22:21:21.729 13022 13022 E AndroidRuntime: 	... 15 more
```

--- a/app/src/main/java/com/stevesoltys/seedvault/ui/recoverycode/RecoveryCodeViewModel.kt
+++ b/app/src/main/java/com/stevesoltys/seedvault/ui/recoverycode/RecoveryCodeViewModel.kt
@@ -61,7 +61,9 @@ internal class RecoveryCodeViewModel(
 
     @Throws(InvalidWordException::class, ChecksumException::class)
     fun validateCode(input: List<CharSequence>): Mnemonics.MnemonicCode {
-        val code = Mnemonics.MnemonicCode(input.toMnemonicChars())
+        check(input.size == WORD_NUM) { "Got ${input.size} words instead of $WORD_NUM" }
+        val trimmedInput = input.map { it.trim() } // white-spaces have a meaning in next line
+        val code = Mnemonics.MnemonicCode(trimmedInput.toMnemonicChars())
         try {
             code.validate()
         } catch (e: WordCountException) {

